clients/fides-js/src/components/fides.css

```diff
@@ -399,12 +414,15 @@ div#fides-consent-content .fides-modal-body {
 }
 
 .fides-modal-footer {
+  position: relative;
   display: flex;
   flex-direction: column;
   z-index: 5;
   background-color: var(--fides-overlay-background-color);
   bottom: 0px;
-  width: var(--fides-overlay-width);
+  width: calc(
```

Comment:
Wondering if this is necessary. It's parent "div.fides-modal-content" already has the overlay width set. So no ´width´ or width set to 100% _should_ have the same result I think.

Reply:
you are correct. This change was necessary _before_ I applied the `box-sizing` to the entire component. Before it was bleeding over with `100%` set. I'll take another look now that it's more standard.